### PR TITLE
ec-440 tx generator

### PR DESCRIPTION
**Pull request by GusevTimofey**

### CODE REVIEW

File: src/main/scala/org/encryfoundation/generator/actors/Broadcaster.scala [hunk 1]
```diff
@@ -1,18 +1,35 @@
...
+import org.encryfoundation.generator.actors.InfluxActor.SendTxsQty
+import scala.concurrent.duration._
+import scala.language.postfixOps
 
-class Broadcaster extends Actor {
+class Broadcaster extends Actor with StrictLogging {
+
+  var sendTxs: Int = 0
+
+  system.scheduler.schedule(5 second, 60 second) {
```
Comment (GRYE) [inline src/main/scala/org/encryfoundation/generator/actors/Broadcaster.scala]: Replace w context

File: src/main/scala/org/encryfoundation/generator/actors/Broadcaster.scala [hunk 2]
```diff
@@ -1,18 +1,35 @@
...
 import org.encryfoundation.generator.utils.NetworkService
+import org.encryfoundation.generator.GeneratorApp._
+import org.encryfoundation.generator.actors.InfluxActor.SendTxsQty
+import scala.concurrent.duration._
+import scala.language.postfixOps
 
-class Broadcaster extends Actor {
+class Broadcaster extends Actor with StrictLogging {
+
+  var sendTxs: Int = 0
```
Comment (GRYE) [inline src/main/scala/org/encryfoundation/generator/actors/Broadcaster.scala]: DO we really need it?

File: src/main/scala/org/encryfoundation/generator/actors/UtxoObserver.scala [hunk 3]
```diff
@@ -2,38 +2,58 @@ package org.encryfoundation.generator.actors
...
+    usedUtxsos = TreeSet.empty
+  }
 
   override def receive: Receive = {
     case RequestUtxos(qty) =>
       val takeQty: Int =
         if (qty < 0) pool.size
         else if (qty <= pool.size) qty
         else pool.size
+      //        qty match {
```
Comment (GRYE) [inline src/main/scala/org/encryfoundation/generator/actors/UtxoObserver.scala]: remove comment

File: src/main/scala/org/encryfoundation/generator/actors/UtxoObserver.scala [hunk 4]
```diff
@@ -2,24 +2,32 @@ package org.encryfoundation.generator.actors
...
+class UtxoObserver(host: InetSocketAddress) extends Actor with StrictLogging {
 
   implicit val ec: ExecutionContextExecutor = context.system.dispatcher
 
   var pool: Map[String, Box] = Map.empty
+  var usedUtxsos: TreeSet[String] = TreeSet.empty
 
   val utxosRequest: Cancellable = context.system.scheduler
-    .schedule(initialDelay = 5 seconds, interval = settings.nodePollingInterval seconds)(fetchUtxos())
+    .schedule(initialDelay = 20 seconds, interval = settings.nodePollingInterval seconds)(fetchUtxos())
```
Comment (Bromel777) [inline src/main/scala/org/encryfoundation/generator/actors/UtxoObserver.scala]: Move initialDelay to settings

File: src/main/scala/org/encryfoundation/generator/actors/Broadcaster.scala [hunk 5]
```diff
@@ -1,18 +1,22 @@
...
+import scala.language.postfixOps
 
-class Broadcaster extends Actor {
+class Broadcaster extends Actor with StrictLogging {
 
   override def receive: Receive = {
-    case Broadcaster.Transaction(tx) => settings.peers.foreach { NetworkService.commitTransaction(_, tx) }
-  }
+    case Broadcaster.Transaction(tx) => settings.peers.foreach {
+      NetworkService.commitTransaction(_, tx) }
```
Comment (GRYE) [inline src/main/scala/org/encryfoundation/generator/actors/Broadcaster.scala]: Format it please

**Review by GRYE — APPROVED** [review]

**Review by Bromel777 — APPROVED** [review]

**Review by kapinuss — APPROVED** [review]

**Pull request merged**